src/com/ceco/nougat/gravitybox/ModVolumePanel.java

```diff
@@ -119,11 +119,20 @@ protected void afterHookedMethod(final MethodHookParam param) throws Throwable {
             XposedBridge.hookAllConstructors(classVolumePanelCtrl, new XC_MethodHook() {
                 @Override
                 protected void afterHookedMethod(final MethodHookParam param) throws Throwable {
-                    String[] titles = (String[]) XposedHelpers.getObjectField(param.thisObject, "mStreamTitles");
-                    int idx = AudioManager.STREAM_NOTIFICATION;
-                    if (titles.length > idx && (titles[idx] == null || titles[idx].trim().isEmpty())) {
-                        Context ctx = (Context) XposedHelpers.getObjectField(param.thisObject, "mContext");
-                        titles[idx] = Utils.getGbContext(ctx).getString(R.string.notification_stream_name);
+                    if (Utils.isSamsungRom())
+                    {
+                       int[] titles = (int[]) XposedHelpers.getObjectField(param.thisObject, "STREAMTITLES");
+                       int idx = AudioManager.STREAM_NOTIFICATION;
+                       if (titles.length > idx && (titles[idx]==0)) {
+                          titles[idx] = (R.string.notification_stream_name);

```

Comment:
This won't work. Resource ID does not exist in hooked process. Only in GravityBox context. You would have to inject this resource with fake resource ID into process resources.